core/src/main/java/org/pegdown/ParserWithDirectives.java

```diff
@@ -162,7 +162,7 @@ public Node parseBlockContents(Var<Integer> markerLength, StringBuilderVar block
         // recursively parse inner contents
         Context<Object> context = getContext();
         Integer length = markerLength.get();
-        String contents = block.getString();
+        String contents = "\n" + block.getString() + "\n";
```

Comment:
Wouldn't this have impact on the rendering if we implemented a container block that behaves like "```" like 

```
@@@console
> println("hello")
@@@
```